docs/contributor-docs/v9-upgrade-guide.md

```diff
@@ -64,7 +70,7 @@ For more information, click the name of the component to see its full documentat
 ### Removed packages and scripts
 
 - InstUI-CLI's `create-app`, `create-component` and `create-package` commands: Just follow the instructions in the [usage](#usage) docs.
-- The `template-component`, `template-app` and `template-app` packages have been removed.
+- The `template-component`, `template-package` and `template-app` packages have been removed.
 
 ### Removal of deprecated legacy reference attributes from components
 
```

Comment:
line 77: '...will be deleted in v9'. Isnt this v10? it would be good to list the exact changes here.